common/utils/src/main/resources/error/error-classes.json

```diff
@@ -31,6 +31,15 @@
     ],
     "sqlState" : "42702"
   },
+  "AMBIGUOUS_COLUMN_REFERENCE" : {
+    "message" : [
+      "Column <name> is ambiguous. It's because you joined several DataFrame together, and some of these DataFrames are the same.",
```

Comment:
Can we reuse `AMBIGUOUS_REFERENCE (42704)`?

and I guess another cause maybe dataframe creation with duplicated column names:
```
scala> val df = Seq((1,2), (3,4)).toDF("a","a")
val df: org.apache.spark.sql.DataFrame = [a: int, a: int]
```

Reply:
It's different. This is for found more than one matching dataframes, but `AMBIGUOUS_REFERENCE` is for found more than one matching attribute from candidates.

Reply:
sounds good.